lib/datasource/metadata.js

```diff
@@ -33,6 +33,7 @@ const manualChangelogUrls = {
       'https://firebase.google.com/support/release-notes/admin/python',
     requests:
       'http://docs.python-requests.org/en/master/community/updates/#release-and-version-history',
+    wagtail: 'http://docs.wagtail.io/en/v2.6/releases/index.html',
```

Comment:
Won't this URL be out of date once the v2.7 release is out? I looked for a way to remove the version from the URL but it doesn't seem to work.

Reply:
@rarkins you're right. Maybe it's better to use this link instead then: https://github.com/wagtail/wagtail/tree/master/docs/releases